.circleci/config.yml

```diff
@@ -440,3 +465,18 @@ jobs:
       - run:
           name: Run task
           command: runner -task << parameters.task_name >> -timeout << parameters.timeout >>
+
+commands:
+  copy-merged-cov-to-host:
+    parameters:
+      file_to_copy:
+        description: Path to the file where the cov file will be copied from host machine.
+        type: string
+      copy_to_dir:
+        description: Path to the directory where the cov file will be copied to on the host machine.
+        type: string
+    steps:
+      - run:
+          command: |
+            docker-compose -f docker-compose.yml --project-name api-unit-tests exec api chmod -R 777 "<< parameters.file_to_copy >>"
```

Comment:
Could probably make this more restrictive but given it's just for the tests it should be fine.